src/dynamics/ergodic/add_circle.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2022 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import measure_theory.group.add_circle
+import dynamics.ergodic.ergodic
+import measure_theory.covering.density_theorem
+import data.set.pointwise.iterate
+
+/-!
+# Ergodic maps of the additive circle
+
+This file contains proofs of ergodicity for maps of the additive circle.
+
+# Main definitions:
+
+ * `add_circle.ergodic_zsmul`: given `n : ℤ` such that `1 < |n|`, the self map `y ↦ n • y` on
```

Comment:
Even though it's a trivial consequence, is it worth adding the nsmul version for convenience?

Reply:
Agreed, this is worth having. Done.